src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -381,6 +381,13 @@ public void ProcessCommand(IExecutionContext context, string line, ActionCommand
 
                 HostContext.SecretMasker.AddValue(command.Data);
                 Trace.Info($"Add new secret mask with length of {command.Data.Length}");
+
+                // Also add each individual line. Typically individual lines are processed from STDOUT of child processes.
+                var split = command.Data.Split(new[] { '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Should we also do `StringSplitOptions.TrimEntries` so we can avoid trimming each item?

Reply:
`TrimEntries` is new in dotnet 5+, i am going to merge the dotnet 6 PR and then add this.